src/common/utils/index.ts

```diff
@@ -10,46 +12,37 @@ export function clamp(value, min, max) {
   return value;
 }
 
-function linearGradient(left, right) {
+function linearGradient(left: string, right: string) {
   return `linear-gradient(to right, ${left}, ${right})`;
 }
 
-export function mapFromWindowsTheme(name, windowsTheme, useGradients) {
-  /* eslint-disable no-unused-vars, @typescript-eslint/no-unused-vars */
+export function mapFromWindowsTheme(
+  name: string,
+  windowsTheme: WindowsTheme,
+  useGradients: boolean
+) {
```

Comment:
I'd keep these unused vars for the sake of visibility - easier to spot which vars aren't used yet in our React95 themes compared to OG win 95 theme object

Reply:
I thought it was ok to remove them because I defined the type here:
https://github.com/arturbien/React95/blob/beta/src/types.tsx#L60-L92

So we do have a reference of the entire object.